### DISCUSSION TITLE
Failed to delete record. Make sure that the record is not part of a required relation reference.

### DISCUSSION THREAD

**Discussion by faustinoloeza** [pocketbase/pocketbase#286] [Q&A]
I am testing pocketbase and is the 3er time that this happen(Different servers(Debian 11 Linode)), 
I just test an instance of pocketbase and after a few days I am not able to use the instance anymore.
I am not able to create, delete or update any collections.
When I try to create -> error "Something went wrong while processing your request."
When I try to delete -> error "Failed to delete record. Make sure that the record is not part of a required relation reference."
When I try to update -> error "Failed to update record."
I  dont know is I am doing something wrong or any one has this issue too.

pocketbase v0.3.3 I will test with the new version v0.3.4 asap.

**Comment by ganigeorgiev** [comment]
Could you post here the API response errors from the browser's "Inspect tool > Network tab"?

You could also start the `pocketbase` executable with `--debug` for more detailed console errors OR find the failed request(s) in the "Admin UI > Logs".

**Reply by faustinoloeza** [reply]
I try to update.
Response:
{"code":400,"message":"Failed to update record.","data":{}}

Form Data
fecha_de_publicacion: 2022-08-31 12:00:00.000
idioma: sdd
image: tbM1jA50tDx7GoFBCmtCPQsCxdikbRCj.jfif
pdf: mE8Ydo6cX7jGfIB7HwIqkAAnFlyhRjd6.PDF
titulo: sdsd

I try to DELETE
Response:
{"code":400,"message":"Failed to delete record. Make sure that the record is not part of a required relation reference.","data":{}}

Request URL: http://139.144.40.79:8000/api/collections/bKDnY9NUOeMxo1H/records/01hxfpsPQNO3bSL
Request Method: DELETE
Status Code: 400 Bad Request
Remote Address: 139.144.40.79:8000
Referrer Policy: strict-origin-when-cross-origin

![image](https://user-images.githubusercontent.com/6463553/183777195-b45a1709-4d88-4057-937c-eabedcffbc67.png)

**Reply by ganigeorgiev** [reply]
I'm not sure how to reproduce the issues you have encountered.

How do you perform the requests?
Does it fail when you try to create/update/delete from the Admin UI?
Could you post similar screenshots of the logs related to the other failed requests, not just 404?
Do you have any colleciton API rules set?
What is your collections structure?

Try starting the executable with `--debug` and check the console output when you make requests to see whether a more detailed app error is thrown.
